File: lua/refactoring/tests/refactor/106/go/struct-return/extract.expected.go
package main

import "fmt"

type foobar struct {
   poggers int
}

func (f *foobar) foo(a int, test int) INPUT_RETURN_TYPE {
    test_other := 1
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(f.poggers, idx, a)
    }
    return test_other
}

func (f *foobar) simple_function(a int) (int, int) {
    var test int = 1
    test_other := f.foo(a, test)


    return test, test_other
}

File: lua/refactoring/tests/refactor/106/lua/multiple-extract/extract.expected.lua

local function foo_bar(a, test, test_other)
    for idx = test - 1, test_other do
        print(idx, a)
    end
end


function simple_function(a)
    local test = 1
    local test_other = 11
    foo_bar(a, test, test_other)

    for idx = test - 1, test_other do
        print(idx, test)
    end

    foo_bar(a, test, test_other)
end
